src/areaSearch/components/AreaSearchApplicationEdit.js

```diff
@@ -91,14 +97,39 @@ class AreaSearchApplicationEdit extends Component<Props, State> {
 
     initialize(getInitialAreaSearchEditForm(areaSearch));
   }
+  
+  handleFileAdded = (e: Event) => {
+    const {uploadAttachment, areaSearch, setAreaSearchAttachments} = this.props;
+    const currentFiles = areaSearch?.area_search_attachments || [];
+    const file = e.target.files[0];
```

Comment:
Does this support adding multiple files at once, or is it going to run once for each file?